test/tests/ver-1ib/ver-1ib.i

```diff
@@ -0,0 +1,123 @@
+k_b = '${units 1.380649e-23 J/K}' # Boltzmann constant (from PhysicalConstants.h - https://physics.nist.gov/cgi-bin/cuu/Value?r)
```

Comment:
This input file could be deleted because we can leverage the ver-1ia input file.